snowpack/src/config.ts

```diff
@@ -594,7 +594,7 @@ function normalizeAlias(config: SnowpackConfig, cwd: string, createMountAlias: b
       replacement.startsWith('/')
     ) {
       delete cleanAlias[target];
-      cleanAlias[addTrailingSlash(target)] = addTrailingSlash(path.resolve(cwd, replacement));
+      cleanAlias[target] = addTrailingSlash(path.resolve(cwd, replacement));
```

Comment:
This change looks fine to me! I agree that aliasing should work for files as well as folders. If I had to guess as to why this behavior was here in the first place, I’d guess it was carried over from our `mount` configuration, which by its nature _must_ enforce that only directories be mounted. Which doesn’t make sense for `alias`.

Overall I’m +1 in favor of this change, but as this would change behavior for some existing setups, I’d also like to suggest on [util.ts#L295](https://github.com/snowpackjs/snowpack/blob/master/snowpack/src/util.ts#L295): removing the `removeTrailingSlash(from)` check, which I assume is undoing the operation here in `config.ts`.

Additionally, while I’d _like_ to add some new error message like (`Could not find [myalias]. Did you mean [myalias]/?`), this logic takes place in a greater context of us trying several things to resolve an import and failing silently. So I can’t really think of a place where surfacing an error message would make any sense. So in the case of someone relying on this behavior, I want to say that the error message itself should be clear enough on its own.

But backing up to the original intent of this PR, I don’t even think we have to worry about that because rarely will someone try and import a directory anyway.